### ISSUE TITLE
Show warning to user on GUI when encounter corrupted data

### ISSUE THREAD

**Issue by matchastack** [AY2425S1-CS2103-F10-2/tp#112]

**Comment by matchastack** [comment]
This is not a simple task as it involves multiple fundamental change to how 'corrupt' data are handled. 
Messages only appear on the GUI when command is executed and is received either through success return message, or exception.

Given the nature of corrupt data is an error, in order to continue parsing subsequent data, the exception has to be caught and skipped (i.e. `try {...} catch (...) { continue; }`) instead of thrown. This means that the UI will never receive an exception and hence impossible to show a message. If the exception were to be thrown, then it would prevent the parsing of subsequent data. 

In short, this presents a dilemma between the ability to keep uncorrupted data v.s. verbosity in GUI. Given that the warning has already been logged, keeping uncorrupted data to minimise data loss is a more critical feature. Therefore, this issue will be postponed till further notice.
